src/common/PopoverItem.js

```diff
@@ -22,10 +29,34 @@ class PopoverItem extends Component {
         };
     }
 
+    /**
+     * Popover toggle
+     *
+     * @param  {popOverOpen}
+     * @param  isActive {bool} - from parent component Attend received as props
+     * @param  onClick {func} - from parent component received as props
+     */
```

Comment:
Please move the documentation here somewhere else, since it's not related to params this function receives.